keps/sig-cloud-provider/20190729-building-without-in-tree-providers.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Building Kubernetes Without In-Tree Cloud Providers
+authors:
+  - "@janedoe"
+owning-sig: sig-xxx
+participating-sigs:
+  - sig-cloud-provider
+  - sig-release
+reviewers:
+  - "@cheftako"
+  - "@andrewsykim"
+  - "@stephenaugustus"
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2019-07-29
+last-updated: 2019-07-29
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
```

Comment:
I think this is actually implementable? It is prototyped in a functional form modulo CI and any tooling to keep it functional, and has been previously discussed a few times with positive feedback, however I'm unsure if that's the correct state.

Reply:
sgtm but would be good to detail some of the implementation details here before doing that

Reply:
switched to implementable, updating with more details.

Reply:
+1; provisional means we agree its something we need to do. Implementable means we agree on how to do it.